ReactAndroid/src/main/java/com/facebook/react/uimanager/events/ContentSizeChangeEvent.java

```diff
@@ -16,12 +18,26 @@
   private final int mWidth;
   private final int mHeight;
 
+  /**
+   * See {@link Event#Event(int)}.
+   *
+   * @param viewTag
+   * @param width
+   * @param height
+   */
+  @Deprecated
```

Comment:
I don't think this scales well.
We will end up with ecosystem with half events done properly and another half done deprecated way.